custom-namespaces.yaml

```diff
@@ -342,6 +342,10 @@ firefox_accounts:
       tables:
         - channel: release
           table: mozdata.firefox_accounts.fxa_users_first_seen
+    fxa_users_services_first_seen_table:
+      type: table_view
+      tables:
+        table: mozdata.firefox_accounts_derived.fxa_users_services_first_seen
```

Comment:
views defined in `view.sql` automatically should get created in both projects:
- moz-fx-data-shared-prod
- mozdata

Since all the other view for fxa are referencing mozdata I chose to do the same here for consistency. This is referencing a view.

Reply:
In that case I think the view would be `mozdata.firefox_accounts.fxa_users_first_seen`
The view you're linking doesn't exist: `table: mozdata.firefox_accounts_derived.fxa_users_services_first_seen`

Reply:
In that case, lloking at the rest of the definitions in the file, the view to add would be `mozdata.firefox_accounts.fxa_users_services_first_seen`
The one you are adding doesn't exist...